src/otx/v2/adapters/datumaro/adapter/anomaly_dataset_adapter.py

```diff
@@ -7,25 +7,25 @@
 from __future__ import annotations
 
 import os
-from typing import Dict, List, Optional
+from typing import Dict, List, Optional, Union
 
 import cv2
 import numpy as np
 from datumaro.components.annotation import Bbox as DatumBbox
 from datumaro.components.annotation import Label as DatumLabel
 from datumaro.components.annotation import Mask as DatumMask
 from datumaro.components.dataset import Dataset as DatumDataset
+from datumaro.components.annotation import Categories as DatumCategories
 
 from otx.v2.adapters.torch.modules.utils.mask_to_bbox import mask2bbox
-from otx.v2.api.entities.datasets import DatasetEntity
 from otx.v2.api.entities.id import ID
 from otx.v2.api.entities.label import LabelEntity
 from otx.v2.api.entities.label_schema import LabelSchemaEntity
 from otx.v2.api.entities.subset import Subset
-from otx.v2.api.entities.utils.segmentation_utils import create_annotation_from_segmentation_map
 
 from .datumaro_dataset_adapter import DatumaroDatasetAdapter
 
+LabelInformationType = Dict[str, Union[List[LabelEntity], List[DatumCategories]]]
```

Comment:
This can be imported from datumaro dataset adapter.

Reply:
Done. https://github.com/openvinotoolkit/training_extensions/pull/2631/commits/1cbcdcd953906dadb225f5d365962c2dc9ba8467